src/tractusx_sdk/dataspace/controllers/connector/v0_9_0/edr_controller.py

```diff
@@ -0,0 +1,44 @@
+#################################################################################
+# Eclipse Tractus-X - Software Development KIT
+#
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the
+# License for the specific language govern in permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+from .dma_controller import DmaController
+from ..mixins import CreateControllerMixin, GetAllControllerMixin, DeleteControllerMixin
+from ....models.connector.v0_9_0 import ContractNegotiationModel
+
+
+class EdrController(CreateControllerMixin, GetAllControllerMixin, DeleteControllerMixin, DmaController):
+    """
+    Concrete implementation of the EdrController for the Connector v0.9.0 Data Management API.
+
+    This class overrides the create method in order to ensure the correct class types are used, instead of the generic ones.
+    """
+
+    endpoint_url = "/v3/edrs"
+
+    def get_data_address(self, oid: str, **kwargs):
+        return self.adapter.get(url=f"{self.endpoint_url}/{oid}/dataaddress", **kwargs)
```

Comment:
```suggestion
    def get_data_address(self, oid: str, auto_refresh : bool = True, **kwargs):
        return self.adapter.get(url=f"{self.endpoint_url}/{oid}/dataaddress?auto_refresh={auto_refresh}", **kwargs)
```

Could we include the `auto_refresh` parameter? I believe it’s hidden in the Swagger docs, and by default, it’s set to true.

https://eclipse-tractusx.github.io/api-hub/tractusx-edc/0.9.0/control-plane/swagger-ui/#/Control%20Plane%20EDR%20Api/getEdrEntryDataAddressV3

Reply:
The way the adapter is implemented, you can already pass the `auto_refresh` query parameter to the request by doing the following call:

```python
get_data_address("200", params={"auto_refresh": True):
```

The `params` key will be picked up by the `kwargs`, and passed on to the adapter request, which then passes it to the session request itself.

Do you want to define it more explicitelly?

Reply:
I understand now, it didn’t occur to me to try it that way. It’s fine by me if you want to leave it like this. 
Thanks for the clarification.